cmd/sonobuoy/app/retrieve.go

```diff
@@ -31,11 +30,7 @@ import (
 var (
 	prefix        = filepath.Join("tmp", "sonobuoy")
 	defaultOutDir = "."
-)
-
-var (
-	cpKubecfg   Kubeconfig
-	cpNamespace string
+	namespace     string
```

Comment:
please put this in a struct or call it `cpNamespace`. this namespace is shared by the entire package.